node_modules/gh-events/lib/api.js

```diff
@@ -300,6 +300,87 @@ var updateEvent = module.exports.updateEvent = function(ctx, id, update, callbac
 };
 
 /**
+ * Update the organisers of an event
+ *
+ * @param  {Context}        ctx                     Standard context containing the current user and the current app
+ * @param  {Number}         id                      The id of the event to update
+ * @param  {Object}         update                  The updates to persist. Keys that are numbers are considered to be references to users, anything else is considered to be a plain-text organiser
+ * @param  {Function}       callback                Standard callback function
+ * @param  {Object}         callback.err            An error object, if any
+ * @param  {Event}          callback.event          The updated event
+ */
+var updateEventOrganisers = module.exports.updateEventOrganisers = function(ctx, id, update, callback) {
+    update = update || {};
+
+    var validator = new Validator();
+    validator.check(null, {'code': 401, 'msg': 'Only authenticated users can update events'}).isLoggedInUser(ctx);
+    validator.check(id, {'code': 400, 'msg': 'A valid event id must be provided'}).isInt();
+    validator.check(_.keys(update).length, {'code': 400, 'msg': 'At least 1 update value must be provided'}).min(1);
+    _.each(update, function(val, user) {
+        update[user] = GrasshopperUtil.getBooleanParam(update[user]);
+        validator.check(null, {'code': 400, 'msg': 'A valid organiser change must be a boolean value'}).isBoolean(update[user]);
+    });
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Ensure the event exists
+    getEvent(ctx, id, function(err, event) {
+        if (err) {
+            return callback(err);
+        }
+
+        // Get the user ids, if any, from the update
+        var userIds = _.chain(update)
```

Comment:
Wouldn't it be cleaner to split off the actual user ids from the other organisers into separate variables as early on as possible to get cleaner method signatures?